### PR TITLE
Prepare README for v0.5.0 release

### PR DESCRIPTION
**Pull request by adzialocha**
Closes: https://github.com/p2panda/aquadoggo/issues/526

### CODE REVIEW

**Comment by adzialocha** [comment]
This looks really amazing to me now! Thank you @sandreae for all of these good ideas for the README.

**Comment by adzialocha** [comment]
This I stumbled upon, what do you think? It's right at the beginning but directly diverges to other projects, I think that can be a bit confusing (people want to get started with aquadoggo and not with fishy)? Could the section just move down, or at least somehow the links part?

**Comment by sandreae** [comment]
Yeh, totally agree that it's weird to link out to other resources right away. I had the idea to add a new section near the end of the README titled `Resources` and here we can list fishy, send-to-node, tutorials etc... Then we don't need to link to them directly in the section you highlighted.

**Comment by sandreae** [comment]
I also want to add a section showing some GraphQL query examples.

File: README.md [hunk 1]
```diff
@@ -43,65 +43,189 @@
...
 - Discovers other nodes in local network and internet.
-- Replicates data with other nodes.
+- Establishes peer-to-peer connections via UDP holepunching or via relays.
+- Replicates data efficiently with other nodes.
+
+## Who is this for?
+
+`aquadoggo` might be interesting for anyone who wants to participate in a p2p network. This could be as a node maintainer, an application developer or simply someone wanting to learn more about p2p networking in a hands-on fashion.
+
+No programming experience is needed to deploy a node on your own machine, and you can even experiment with creating your own data schemas using, publishing and replicating data, and then querying it again using the GraphQL playground.
```
Comment (adzialocha) [inline README.md:67]: "using"

File: README.md [hunk 2]
```diff
@@ -43,65 +43,297 @@
...
+If you are not working with Rust you can create FFI bindings from the `aquadoggo` crate into your preferred programming language. Dealing with FFI bindings can be a bit cumbersome and we do not have much prepared for you (yet), but check out our [Meli](https://github.com/p2panda/meli/) Android project as an example on how we dealt with FFI bindings for Dart / Flutter.
+
+## Resources
+
+- 🐬 Deploy your own `aquadoggo` following the [tutorial](https://p2panda.org/tutorials/aquadoggo)
+- 🛠️ Create your own schemas using [`fishy`](https://github.com/p2panda/fishy)
+- 🛼 Open the GraphQL playground in your browser, served under `http://localhost:2020/graphql`
+- 📖 Try the [mushroom app tutorial](https://p2panda.org/tutorials/mushroom-app)
+- 🔬 Manually publish JSON encoded data to a node [`send-to-node`](https://github.com/p2panda/send-to-node)
+- 🐼 [Learn more](https://p2panda.org/learn/) about how p2panda works
```
Comment (adzialocha) [inline README.md:322]: Even though it's true it's confusing to mention JSON here maybe, people might think that p2panda data in general is JSON encoded

File: README.md [hunk 3]
```diff
@@ -43,65 +43,297 @@
...
+A collection query for "events" which includes ordering and filtering as well as selecting some meta fields. Here only events between the specified dates and with a title containing the string 'funtastic' will be returned, they will be arranged in ascending chronological order:
+```graphql
+{
+  events: all_events_0020aaabb3edecb2e8b491b0c0cb6d7d175e4db0e9da6003b93de354feb9c52891d0(
+    first: 20
+    orderBy: "happening_at"
+    orderDirection: ASC
+    filter: {
+      title: { contains: "funtastic" }
+      happening_at: { gte: 2023-08-31, lte: 2023-09-14 }
```
Comment (adzialocha) [inline README.md:201]: Think the values need to be an array

Comment (sandreae) [reply README.md:258]: Which values?

Comment (sandreae) [reply README.md:258]: The `happening_at` values need to be unix timestamps

**Comment by sandreae** [comment]
Aaaaaand actually I recon we need to beef up our doc strings in `lib.rs`, they still contain something very similar to the old README.

**Comment by sandreae** [comment]
> Oh, that's pretty cool, examples in the included README are also run during tests (although it fails right now).

I had do disable this feature for now cos there is required code setup which we can't do in the README :sweat_smile:

**Review by sandreae — APPROVED** [review]

**Pull request merged**